services/cosmos-db/mgmt/2015-04-08/documentdb/client.go

```diff
@@ -34,28 +34,18 @@ type BaseClient struct {
 	autorest.Client
 	BaseURI        string
 	SubscriptionID string
-	Filter         string
-	Filter1        string
-	DatabaseRid    string
-	CollectionRid  string
-	Region         string
 }
 
 // New creates an instance of the BaseClient client.
-func New(subscriptionID string, filter string, filter1 string, databaseRid string, collectionRid string, region string) BaseClient {
-	return NewWithBaseURI(DefaultBaseURI, subscriptionID, filter, filter1, databaseRid, collectionRid, region)
+func New(subscriptionID string) BaseClient {
```

Comment:
@marstr This is the behavior of parameter location, this creates incorrect parameter in the client

Reply:
Thanks for the heads up. It does look better this way.